com.teamspeak.TeamSpeak3.yaml

```diff
@@ -0,0 +1,60 @@
+app-id: com.teamspeak.TeamSpeak3
+tags:
+  - proprietary
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: teamspeak3
+finish-args:
+  - --socket=x11
+  - --socket=wayland
```

Comment:
it's either `x11` or `fallback-x11` + `wayland`. Also does it suddenly support Wayland? The other package doesn't have it.

Reply:
Okay, let's drop this for now, will do that sometimes later.

But TS3 is a casual QT application and should work fine with that.